### ISSUE TITLE
platter & magic_carpet not rendered

### ISSUE THREAD

**Issue by not-available-username** [organicmaps/organicmaps#2873]
`Tag:aerialway=platter` and `Tag:aerialway=magic_carpet` are not rendered in the map

**Comment by pastk** [comment]
Looks like some `aerialway` types were added by mistake as e.g. a `piste:lift|magic_carpet;608;`

**Comment by not-available-username** [comment]
As a side note, I think that many things are not displayed until you zoom very much and can't have a wide view of the area, with the main elements and landmark. I guess that displaying many elements could be an issue in metropolitan areas, but the outdoor on OM looks a bit like a desert 😅 

Also, is `Tag:natural=grassland` rendered over `Tag:amenity=parking`?

**Comment by pastk** [comment]
> As a side note, I think that many things are not displayed until you zoom very much and can't have a wide view of the area, with the main elements and landmark. I guess that displaying many elements could be an issue in metropolitan areas, but the outdoor on OM looks a bit like a desert sweat_smile
> 

For area features there is this idea: https://github.com/organicmaps/organicmaps/issues/2856

> Also, is `Tag:natural=grassland` rendered over `Tag:amenity=parking`?

Its probably this case: https://github.com/organicmaps/organicmaps/issues/2170

**Comment by not-available-username** [comment]
> For area features there is this idea: #2856 

I like it, the map looks better and is easier to find elements on it!
> 
> Its probably this case: #2170

I think so, nice to see a proposed fix, maybe is not perfect for everything, but in an improvement in most cases.

**Comment by not-available-username** [comment]
> What about search? Did it work?

Yes, if you know the name 😅 I spotted it with binoculars but didn't know the name

**Comment by not-available-username** [comment]
added some elements, in general the map has very few differentiation on land use, everything looks similar.

**Comment by d4f5409d** [comment]
Cemeteries are displayed and I can search for it, however, only cemeteries who have "name=" tag on it

**Comment by aronkvh** [comment]
Tag: club=sport is also missing for me

**Comment by not-available-username** [comment]
man_made=beacon is not rendered

**Comment by AndreiLazarev** [comment]
bare_rock please!

![2023-06-19 17 36 04](https://github.com/organicmaps/organicmaps/assets/91606966/7ac4cb69-2fb1-40f5-a292-d78299c7882c)

<img width="795" alt="Снимок экрана 2023-06-19 в 17 37 51" src="https://github.com/organicmaps/organicmaps/assets/91606966/337114cb-fb72-4714-860c-f08d6af41c32">

**Comment by d4f5409d** [comment]
natural=beach isn't showing up on the map either

**Comment by pastk** [comment]
> bare_rock please!

We had `natural=bare_rock` for quite some time already.. I saw them just recently in the sea :)
Could you please post coordinates?

**Comment by snooppr** [comment]
Gullies are not displayed on the map and are not found when searching.  Add
natural = [valley](https://wiki.openstreetmap.org/wiki/RU:Tag:natural=valley?uselang=ru)

**Comment by Misalf-git** [comment]
`natural=sand`
`surface=sand`

**Comment by pastk** [comment]
> `natural=sand` `surface=sand`

Note that `surface=sand` is supposed to be used as an attribute of some "main tag". It doesn't make sense standalone.
OM supports `natural=beach + surface=sand` already (but its visible on a long tap only).

**Comment by pastk** [comment]
- `natural=sand` is tracked in https://github.com/organicmaps/organicmaps/issues/2931

**Comment by RedAuburn** [comment]
the main other missing sand things i've noticed were playground sandpits, and golf bunkers

**Comment by Misalf-git** [comment]
`athletics=long_jump
leisure=pitch
sport=athletics
surface=sand`

**Comment by AndreiLazarev** [comment]
> > bare_rock please!
> 
> We had `natural=bare_rock` for quite some time already.. I saw them just recently in the sea :) Could you please post coordinates?

Hi! Still haven't bare_rock :(

For example:
1) 61°09′55.6″N 29°43′57.71″E
2) 61°15′11.77″N 30°06′09.87″E

**Comment by pastk** [comment]
> Hi! Still haven't bare_rock :(
> 
> For example:
> 
>     1. 61°09′55.6″N 29°43′57.71″E

Try to long tap :)
![image](https://github.com/organicmaps/organicmaps/assets/18434508/0f9497de-df20-45d4-a29a-530e4aafb106)

I guess you expected its displayed somehow else?
I think the current color should be changed to a more grey dark. Right now its even lighter than the default (empty) map background.

**Comment by AndreiLazarev** [comment]
> > Hi! Still haven't bare_rock :(
> > For example:
> > ```
> > 1. 61°09′55.6″N 29°43′57.71″E
> > ```
> 
> Try to long tap :) ![image](https://private-user-images.githubusercontent.com/18434508/318117624-0f9497de-df20-45d4-a29a-530e4aafb106.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE3NDQwODIsIm5iZiI6MTcxMTc0Mzc4MiwicGF0aCI6Ii8xODQzNDUwOC8zMTgxMTc2MjQtMGY5NDk3ZGUtZGYyMC00NWQ0LWEyOWEtNTMwZTRhYWZiMTA2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI5VDIwMjMwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFjOWUyM2Y4NmUzYTIzYzIwNTZjY2Y5MDEzMmE3YzY1Y2RlNTQ4ZDRjZjdjNGE2MTYwNmQ3ODE1MDliYWE0YTQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.BjRKD-Ym1pIF1mbUnSxfsJLMJtL2PLxOk1v5SEkstos)
> 
> I guess you expected its displayed somehow else? I think the current color should be changed to a more grey dark. Right now its even lighter than the default (empty) map background.

Thank you! Yes, I look and for me it’s just an emptiness on the map. Maybe you can use vector patterns for something like this?

**Comment by pastk** [comment]
Unfortunately OM doesn't support patterns/textures yet
- https://github.com/organicmaps/organicmaps/issues/1415

**Comment by not-available-username** [comment]
the building isn’t displayed if has`tag:tourism=alpine_hut`

**Comment by pastk** [comment]
> the building isn’t displayed if has`tag:tourism=alpine_hut`

could you please provide an example? (coords or osm link)

**Comment by not-available-username** [comment]
> > the building isn’t displayed if has`tag:tourism=alpine_hut`
> 
> could you please provide an example? (coords or osm link)

Sorry, it was my fault, I used the wrong tags on a building, already corrected. Thanks!

**Comment by dvdmrtnz** [comment]
Closing this meta-issue. Please add new POI feature requests in separate issues
